docs/DafnyRef/Statements.md

```diff
@@ -1891,10 +1907,26 @@ depend on any testing framework in the target language.
 Within such methods one might use `expect` statements (as well as `print` statements)
 to insert checks that the target program is behaving as expected.
 
-C) Compiler tests
+C) Debugging
 
-If one wants to assure that compiled code is behaving at run-time consistently with the statically verified code,
-one can use paired assert/expect statements with the same expression:
+While developing a new program, one work style uses proof attempts and runtime tests in combination.
+If an assert statement does not prove, one might run the program with a corresponding expect statement
+to see if there are some conditions when the assert is not actually true. So one might have
+paired assert/expect statements:
+<!-- %no-check -->
+```dafny
+assert _P_;
+expect _P_;
+```
+Once the program is debugged, both statements can be removed.
+Note that it is important that the `assert` come before the `expect`, because
+to the verifier, the `expect` is an `assume`, which would automatically make
```

Comment:
```suggestion
by the verifier, the `expect` is interpreted as an `assume`, which would automatically make
```

I'm not set on this, but it might be an improvement.